src/test/Components/Blueprints/Blueprints.test.tsx

```diff
@@ -11,6 +11,30 @@ import { emptyGetBlueprints } from '../../fixtures/blueprints';
 import { server } from '../../mocks/server';
 import { renderCustomRoutesWithReduxRouter } from '../../testUtils';
 
+vi.mock('@redhat-cloud-services/frontend-components/useChrome', () => ({
+  useChrome: () => ({
+    isBeta: () => true,
+    isProd: () => true,
+    getEnvironment: () => 'prod',
+  }),
+}));
+
+vi.mock('@unleash/proxy-client-react', () => ({
+  useUnleashContext: () => vi.fn(),
+  useFlag: vi.fn((flag) => {
+    switch (flag) {
+      case 'image-builder.firstboot.enabled':
+        return true;
+      case 'image-builder.snapshots.enabled':
+        return true;
+      case 'image-builder.import.enabled':
+        return true;
+      default:
+        return false;
+    }
+  }),
+}));
+
```

Comment:
A late FYI - mocking chrome and unleash should no longer be needed within test files. It's included in the test setup file now. https://github.com/osbuild/image-builder-frontend/blob/f60e80b4e88fbb9604b213818bfa5acd7404c0ba/src/test/setup.ts#L18-L64

We can remove the local mocks in a small follow-up

Reply:
Okay, sure! If we can do that in a follow-up, that would be nice. I will start working on it, and open a PR once we merge this. :)